l10n_br_account_product/models/account_invoice.py

```diff
@@ -492,6 +492,45 @@ def action_date_assign(self):
                 invoice.write(result['value'])
         return True
 
+    @api.multi
+    def button_reset_taxes(self):
+        result = super(AccountInvoice, self).button_reset_taxes()
+        ait = self.env['account.invoice.tax']
+        for invoice in self:
+            invoice.read()
+            costs = []
+            company = invoice.company_id
```

Comment:
@lfdivino Please raise an excetion UserError if the tax are not filled in company form